packages/navie/src/commands/explain-command.ts

```diff
@@ -96,6 +97,10 @@ export default class ExplainCommand implements Command {
     if (hasChatHistory) {
       for (const e of await this.memoryService.predictSummary(chatHistory))
         this.interactionHistory.addEvent(e);
+
+      for (const e of getMostRecentMessages(chatHistory)) {
+        this.interactionHistory.addEvent(e);
+      }
```

Comment:
Is this problematic in that it doesn't consider or affect the token limit?

Reply:
Even in SWE bench it never happened except for when there were huge stack traces.